dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -533,7 +533,11 @@ Block DeltaMergeStore::addExtraColumnIfNeed(
     return std::move(block);
 }
 
-DM::WriteResult DeltaMergeStore::write(const Context & db_context, const DB::Settings & db_settings, Block & block)
+DM::WriteResult DeltaMergeStore::write(
+    const Context & db_context,
+    const DB::Settings & db_settings,
+    Block & block,
+    RegionID region_id)
```

Comment:
We also need to print out `applied_index`. So better changed to a context struct.